accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
 different sets of document types available in Chile. By default, all the sales journals created will
 use documents. 
   
-.. image:: media/Chile10.png
+.. image:: media/Chile41.png
    :align: center
    :alt: Document type configuration on Journal.
-
-Sequences
-*********
-
-When a Journal is created we have two options with document types sequences. 
-
-1. Create new sequences: In this case, Odoo will create new sequences for the most common document
-   types. 
-2. Use existent sequence: If the sequence should be shared between several journals, the user can
-   add manually sequences previously created.  
-
-.. image:: media/Chile11.png
```

Comment:
Is this file still used somewhere?

Reply:
No, it has been removed in the last commt